tests/hash_table_test.c

```diff
@@ -1084,7 +1084,7 @@ static int s_test_hash_churn_fn(struct aws_allocator *allocator, void *ctx) {
             int was_present;
             err_code = aws_hash_table_remove(&hash_table, e->key, NULL, &was_present);
             ASSERT_SUCCESS(err_code, "Unexpected failure removing element");
-            if (i == 0 && entries[i - 1].key == e->key && entries[i - 1].is_removed) {
+            if (i != 0 && entries[i - 1].key == e->key && entries[i - 1].is_removed) {
```

Comment:
oh man ... this test
I don't know what it's trying to do
but a few lines up it's calling [rand()](https://en.cppreference.com/w/c/numeric/random/rand) which returns a random **integer** and checking whether it's `< 0.05` (floating point) 😬. I think they expect that to pass 5.0% of the time, but in reality it's going to pass 0.003% of the time.

Reply:
OK, I fixed the test in a separate PR: https://github.com/awslabs/aws-c-common/pull/1172